index.html

```diff
@@ -84,197 +46,249 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
+      <div class="p-notification--information">
+        <p class="p-notification__response">
+          <span class="p-notification__status">Information:</span>
+          MicroStack requires at least 8 GB of RAM and a multi-core processor.
+        </p>
+      </div>
+      <h3>What OS are you developing on?</h3>
+    </div>
 
-          <h4>On Windows and macOS</h4>
+    <div class="row u-equal-height u-sv3">
+      <a id="tab-one" href="#tab-one__content"
+        class="col-small-2 col-medium-2 col-4 p-card p-tab u-align--center u-vertically-center js-tab" role="tab">
+        <img src="https://assets.ubuntu.com/v1/53c3be42-linuc-logo.svg" style="height: 90px; margin-top: 1rem;" alt="">
+        <h4 class="p-heading--five p-tab__title">Linux</h4>
+      </a>
 
-          <p>
-            Install Multipass then start a multipass vm:
-          </p>
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" value="multipass launch --name microkstack-vm --mem 8G --disk 40G" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+      <a id="tab-two" href="#tab-two__content"
+        class="col-small-2 col-medium-2 col-4 p-card p-tab u-align--center u-vertically-center js-tab" role="tab">
+        <img src="https://assets.ubuntu.com/v1/11ecbd2d-2018-logo-windows.svg" style="height: 145px;" alt="Windows">
+      </a>
 
-          <p>
-            Install the MicroStack snap on that multipass vm:
-          </p>
-          <div class="p-code-copyable u-sv3">
-            <input class="p-code-copyable__input" value="multipass exec microkstack-vm -- sudo snap install microstack --classic --edge" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+      <a id="tab-three" href="#tab-three__content"
+        class="col-small-2 col-medium-2 col-4 p-card p-tab u-align--center u-vertically-center js-tab" role="tab">
+        <img src="https://assets.ubuntu.com/v1/236f314d-macos.svg" style="height: 145px;" alt="macOS">
+      </a>
+    </div>
 
-          <h3>Configure networks and OpenStack databases</h3>
+    <div class="row">
+      <div class="col-6 js-tab__content" id="tab-one__content" role="tabpanel" aria-labelledby="tab-one">
+        <ol class="p-stepped-list u-no-margin--left">
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install the microkstack snap</h3>
 
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="sudo microstack.init" value="sudo microstack.init --auto" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" aria-label="sudo snap install microk8s --classic"
```

Comment:
```suggestion
              <input class="p-code-copyable__input"
```
Snap name aside, that is not a label.

(If it wasn’t obvious that the actual label for this field is “Install the microstack snap”, you could use `aria-labelledby="id-of-that-heading"`. But there’s no text separating the heading and the field, so I don’t think that’s necessary.)